frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -201,7 +201,41 @@
     "TASKCFG_ALL_KAFKA_REPLICA_HIGH_WATERMARK_CHECKPOINT_INTERVAL_MS": "{{kafka.replica_high_watermark_checkpoint_interval_ms}}",
     "TASKCFG_ALL_KAFKA_OFFSETS_TOPIC_SEGMENT_BYTES": "{{kafka.offsets_topic_segment_bytes}}",
     "TASKCFG_ALL_KAFKA_LOG_CLEANER_IO_MAX_BYTES_PER_SECOND": "{{kafka.log_cleaner_io_max_bytes_per_second}}",
-    "TASKCFG_ALL_KAFKA_COMPRESSION_TYPE": "{{kafka.compression_type}}"
+    "TASKCFG_ALL_KAFKA_COMPRESSION_TYPE": "{{kafka.compression_type}}",
+
+    {{#kafka.log_flush_interval_ms}}
```

Comment:
I was wondering whether `0` was a Falsey value in the package manager mustache library. Do you know?

Reply:
@nickbp do you know off the top of your head?

Reply:
Not certain but I think it isn't? Could get around this by having the value be a string in config.json with a default value of "". Or if it really needs to default to 0, could add some translation in kafka's Main.java. Can't hurt to check what mustache actually does, though.

Reply:
The reason for asking is that 0 is a valid value for some of these, and if the conditional block would evaluate to false in this case, then it would use the Kafka default instead.

Reply:
I will give it a quick test.

Reply:
Ok, it works in Marathon ok.

Reply:
And, it also works in bootstrap's mustache.
```
log.roll.ms=1000
log.roll.hours=168
log.roll.jitter.ms=0
log.roll.jitter.hours=0
```